function.py

```diff
@@ -0,0 +1,6 @@
+
+def function(n):
```

Comment:
It would be nice to have some comments here to see what the code is expected to do.